### DISCUSSION TITLE
How to update wallet's link as the walletconnect-registry is readonly

### DISCUSSION THREAD

**Discussion by chenyoh** [WalletConnect/walletconnect-monorepo#1414] [Developer Support]
Hi, I'm a developer at ONTO Wallet and I need to modify the configuration on Wallet Connect due to a business change, However, I do not have an account to log in Wallet Connect (because the information of creating ONTO Wallet on Wallet Connect was created by submitting PR on Github), may I ask what should I do in this case?

**Comment by DtechLabs** [comment]
I have some problem. Need update info about our wallet but cloud.walletconnect.com has only one option - create new project

**Comment by chenyoh** [comment]
@chadyj Hello, can you help me solve this problem?

**Comment by looter** [comment]
Hi @chenyoh @DtechLabs  either reply with an email address you have access to after you create a cloud explorer account and let me know what it is and I can make sure that account is associated with the listing.

**Reply by DtechLabs** [reply]
@looter our email  im@plasmapay.com.  We already register new one (Plasma) through cloud explorer. Please remove old wallet `Plasmapay` from registry. Thanks in advanced

**Reply by looter** [reply]
@DtechLabs thanks, I actually went ahead and set PlasmaPay to your account as well. You may want to just update that but I'll leave that up to you. I also sent an email to your address to verify the explorer listing. When you have a moment let's chat there.

**Comment by chenyoh** [comment]
@looter this is our email address  xuwen@onchain.com. Thanks in advance.

**Reply by looter** [reply]
@chenyoh I linked this so you should be able to see the listing on that account. Let me know if you have any questions.

**Comment by Bobablocks** [comment]
Hi, i also wanted to migrate my existing registry as listed here https://github.com/WalletConnect/walletconnect-registry/issues/422 on january 2022 to my cloud registry account which has registered as admin@bobablocks.io.

To be noted, our app has changed its name and several meta data including deep links / app links, support desktop version & support multiple chains.

could you help? @looter Thanks 🙏
